components/dashboard/sections/overview/HostOverviewContent.tsx

```diff
@@ -0,0 +1,197 @@
+      ),
+      amount: {
+        current: data?.host.hostStats.balance,
+        comparison: data?.host.hostStats.comparisonBalance,
+      },
+    },
+    {
+      id: 'received',
+      label: <FormattedMessage defaultMessage="Received" id="z/wUXE" />,
+      helpLabel: <FormattedMessage defaultMessage="Total amount received this period" id="2kY5p5" />,
+      amount: {
+        current: data?.host.hostStats.totalAmountReceived,
+      },
+    },
+    {
+      id: 'spent',
+      label: <FormattedMessage defaultMessage="Spent" id="111qQK" />,
+      helpLabel: <FormattedMessage defaultMessage="Total amount spent this period" id="6ctWuQ" />,
+      amount: {
+        current: data?.host.hostStats.totalAmountSpent,
+      },
+    },
+  ];
+
+  return (
+    <React.Fragment>
+      <HostTodoList />
+
+      <Card>
+        <CardHeader>
+          <CardTitle className="text-xl">
+            <FormattedMessage defaultMessage="Recent Financial Activity" id="BAvsQv" />
+          </CardTitle>
+        </CardHeader>
+        <CardContent className="space-y-3">
+          <Filterbar hideSeparator {...queryFilter} />
+
+          <div className="grid grid-flow-dense grid-cols-1 gap-4 sm:grid-cols-2 md:grid-cols-1 lg:grid-cols-3">
+            {metrics
+              .filter(metric => !metric.hide)
+              .map(metric => (
+                <Metric key={metric.id} {...metric} loading={loading} />
+              ))}
+          </div>
+          <div>
+            <DataTable
+              data-cy="transactions-table"
+              innerClassName="table-fixed text-muted-foreground"
+              columns={columns}
+              data={data?.transactions?.nodes || []}
+              loading={loading}
+              nbPlaceholders={data?.transactions.limit || 5}
+              mobileTableView
+              columnVisibility={{ clearedAt: false, debit: false, credit: false }}
+              getRowId={row => String(row.legacyId)}
+              compact
+              fullWidth={true}
+            />
+          </div>
+          {loading ? (
+            <div className="flex h-8 w-full items-center justify-center">
+              <Skeleton className="h-5 w-40" />
+            </div>
+          ) : data.transactions.totalCount > data.transactions.limit ? (
+            <Button
+              size="xs"
+              className="w-full"
+              variant="ghost"
+              onClick={() => {
+                hostTransactionsQueryFilter.resetFilters(
+                  {
+                    ...(queryFilter.values.context === HostContext.INTERNAL
+                      ? {
+                          account: account.slug,
+                        }
+                      : queryFilter.values.context === HostContext.HOSTED
+                        ? {
+                            excludeAccount: account.slug,
+                          }
+                        : {}),
```

Comment:
This could look a bit better as a const before the return scope.

Reply:
Good call, fixed this